e2e/cypress/integration/scroll/channel_scroll_spec.js

```diff
@@ -33,6 +36,103 @@ describe('Scroll', () => {
+        cy.findByText(firstMessageInCenter).should('exist').and('be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('not.be.visible');
+
+        // # Scroll back to bottom
+        cy.get('div.post-list__dynamic').should('be.visible').scrollTo('bottom', {duration: TIMEOUTS.ONE_SEC});
+
+        // * Verify we scolled to bottom
+        cy.findByText(firstMessageInCenter).should('exist').and('not.be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('be.visible');
+
+        // # Open emoji picker for the last message
+        cy.getLastPostId().then((postId) => {
+            cy.clickPostReactionIcon(postId, 'CENTER');
+        });
+
+        // # Now try to scroll in the center
+        cy.get('#post-list').scrollTo('top', {ensureScrollable: false, duration: TIMEOUTS.ONE_SEC});
+
+        // * Verify that scrolling didnt happen
+        cy.findByText(firstMessageInCenter).should('exist').and('not.be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('be.visible');
+
+        // # Now write first message again so we can add reply
+        cy.postMessage(firstMessageInRhs);
+
+        // # Hit reply to the last message
+        cy.getLastPostId().then((postId) => {
+            cy.clickPostCommentIcon(postId);
+
+            Cypress._.times(3, () => {
+                cy.uiPostMessageQuickly(MESSAGES.HUGE, true);
+            });
+        });
+
+        // # Write the last message in RHS
+        cy.postMessageReplyInRHS(lastMessageInRhs);
+
+        cy.getLastPostId().then((postId) => {
+            cy.get('#rhsContainer').should('exist').within(() => {
+                // # Scroll to top in RHS
+                cy.get('.scrollbar--view').scrollTo('top', {duration: TIMEOUTS.ONE_SEC});
+
+                // * Verify that we can see the top message
+                cy.findByText(firstMessageInRhs).should('exist').and('be.visible');
+                cy.findByText(lastMessageInRhs).should('exist').and('not.be.visible');
+
+                // # Scroll back to bottom
+                cy.get('.scrollbar--view').should('be.visible').scrollTo('bottom', {duration: TIMEOUTS.ONE_SEC});
+
+                // * Verify we scolled to bottom
+                cy.findByText(firstMessageInRhs).should('exist').and('not.be.visible');
+                cy.findByText(lastMessageInRhs).should('exist').and('be.visible');
+
+                // # Open emoji picker for the last message in RHS
+                cy.clickPostReactionIcon(postId, 'RHS_COMMENT');
+
+                // // # Scroll to top in RHS with emoji picker open
+                cy.get('.scrollbar--view').scrollTo('top', {ensureScrollable: false, duration: TIMEOUTS.ONE_SEC});
+
+                // // * Verify that scrolling didnt happen in RHS
+                cy.findByText(firstMessageInRhs).should('exist').and('not.be.visible');
+                cy.findByText(lastMessageInRhs).should('exist').and('be.visible');
```

Comment:
Here RHS is scrolling even thought emoji picker is open, in actual app it isnt happening though